packages/math/package.json

```diff
@@ -23,6 +23,6 @@
   },
   "dependencies": {
     "@yicchi/is-odd": "workspace:*",
-    "@yicchi/is-even": "workspace:*"
+    "@yicchi/is-even": "0.9.0"
```

Comment:
Here it has been updated but the lock file has not been updated. So `pnpm i` will be failed due to specifier' s difference.

Reply:
here is log. https://github.com/YiCChi/front-monorepo/actions/runs/7265929798/job/19796658726